pkg/updater/resultstore/resultstore.go

```diff
@@ -409,20 +422,67 @@ func processGroup(tg *configpb.TestGroup, group *invocationGroup) *updater.Infla
 				if !includeStatus(tg, sar) {
 					continue
 				}
-				// TODO(sultan-duisenbay): sanitize build target and apply naming config
-				var cell updater.Cell
-
-				cell.CellID = invocation.InvocationProto.GetId().GetInvocationId()
-				cell.ID = targetID
 
 				// assign status
 				status, ok := convertStatus[sar.TargetProto.GetStatusAttributes().GetStatus()]
 				if !ok {
 					status = statuspb.TestStatus_UNKNOWN
 				}
+				// TODO(sultan-duisenbay): sanitize build target and apply naming config
+				var cell updater.Cell
+				cell.CellID = invocation.InvocationProto.GetId().GetInvocationId()
+				cell.ID = targetID
 				cell.Result = status
-				if cr := customTargetStatus(tg.GetCustomEvaluatorRuleSet(), sar); cr != nil {
-					cell.Result = *cr
+
+				testResults := getTestResults(sar.ActionProto.GetTestAction().GetTestSuite())
+				for _, testResult := range testResults {
```

Comment:
nit: I think this is worth putting in a separate helper method, so it's clear what processing a group does.

Reply:
+1